source/includes/temp-warn-odbc-catalina.txt

```diff
@@ -0,0 +1,8 @@
+.. (DOCSP-7244) This is a temporary warning that will be removed with BI-2371
+
+.. warning::
```

Comment:
What do you think about changing this to an "important" since we can offer a workaround? I'm 50/50

Reply:
That makes sense to me - it's more of an extra step for some users than a warning